### ISSUE TITLE
Using v0.5.6 for Sanic 20.12.* causes dependency issue

### ISSUE THREAD

**Issue by kirisanth-g** [ashleysommer/sanic-restplus#25]
Currently using Sanic 20.12.3 and am limited form upgrading to v21, thus was trying to use sanic-restplus 0.5.6 as recommended. But I run run into the following dependency issue:
```
There are incompatible versions in the resolved dependencies:
  sanic==20.12.3 (from -r /tmp/pipenv8pobw9kgrequirements/pipenv-s50m307i-constraints.txt (line 17))
  sanic<21,>=18.12.0 (from sanic-plugins-framework==0.9.5->sanic-restplus==0.5.6->-r /tmp/pipenv8pobw9kgrequirements/pipenv-s50m307i-constraints.txt (line 6))
  sanic<21,>=18.12.0 (from sanic-restplus==0.5.6->-r /tmp/pipenv8pobw9kgrequirements/pipenv-s50m307i-constraints.txt (line 6))
  sanic>=18.12 (from sanic-jinja2-spf==0.8.0->sanic-restplus==0.5.6->-r /tmp/pipenv8pobw9kgrequirements/pipenv-s50m307i-constraints.txt (line 6))
  sanic>=21.3 (from sanic-jinja2==0.10.0->sanic-jinja2-spf==0.8.0->sanic-restplus==0.5.6->-r /tmp/pipenv8pobw9kgrequirements/pipenv-s50m307i-constraints.txt (line 6))
```
Any recommendations/fixes?

**Comment by ashleysommer** [comment]
The issue is coming from sanic-jinja2. The latest (v0.10.0) sanic-jinja2 requires sanic 21+.
Try to set your constraints to `sanic-jinja2<0.10`. If I get a chance, I will re-release a v0.5.6 postfix with `sanic-jinja2<0.10` in the dependencies..

**Comment by ashleysommer** [comment]
@kirisanth-g I've released sanic-restplus v0.5.6-post1 that has that fixed.
